include/kernel-version.mk

```diff
@@ -6,10 +6,10 @@ ifdef CONFIG_TESTING_KERNEL
   KERNEL_PATCHVER:=$(KERNEL_TESTING_PATCHVER)
 endif
 
-LINUX_VERSION-5.4 = .155
+LINUX_VERSION-5.4 = .156
```

Comment:
You should change the commit title to "kernel: bump 5.4 to 5.4.156"

Reply:
Typo, thanks for pointing it out.